queries/lmfs/basics/task_outcomes.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+const { query } = require("../../util/query.js");
+const desc = `
+This query computes the breakdown of task outcomes specified
+in update_task calls over the last 30 days.  The query doesn't
+attempt to filter out duplicates. 
+`;
+const argv = require("../../util/args.js").processArgs(desc, {
+  lastNDays: {
+    describe: "Use this value instead of the default",
+    default: 30,
+  },
+});
+const sql = `
+SELECT
+  *
+FROM (
```

Comment:
Complex queries like this might be useful to document as a view for easy querying (and visualization in datastudio/looker/etc).